### ISSUE TITLE
Better Contrast for Malte Buttons

### ISSUE THREAD

**Issue by sarahsporck** [digitalfabrik/integreat-app#2497]
**Is your feature request related to a problem? Please describe.**
Malte buttons do have this incredibly bright-red-danger background. While the text is displayed in some grayish color. This is not only badly accessible, but also hard to read in general.

**Describe the solution you'd like**
Calculate font color from background color, when using colored backgrounds.

**Describe alternatives you've considered**
Use static font colors in build configs for colored backgrounds.

**Additional context**
<img width="300" alt="Bildschirmfoto 2023-10-04 um 09 41 27" src="https://github.com/digitalfabrik/integreat-app/assets/28013368/e3a3e4bf-8d80-4ad1-9e6a-b35432a4bded"><img width="331" alt="Bildschirmfoto 2023-10-04 um 09 44 12" src="https://github.com/digitalfabrik/integreat-app/assets/28013368/996cee3e-4b43-4a34-a1ab-94b5be351117">

**Comment by hauf-toni** [comment]
design/color-proposal can be found [📍 here](https://www.figma.com/file/EiRpMhSqscuaChGdhNVZQ4/MALTE-App?type=design&node-id=54%3A114&mode=design&t=DvW55uNHC9uYQzIX-1) @sarahsporck

**Comment by hauf-toni** [comment]
@sarahsporck I've added a screen design for the feedback view (default + seleocted). Let me know if you have any questions 📭

**Comment by sarahsporck** [comment]
exactly. red is malte and green is aschaffenburg. Both suffer from bad contrast. I also mentioned this in the figma design.

**Comment by f1sh1918** [comment]
> Is this one ready or still in review now @f1sh1918? :) It's a little bit unclear where we ended off. 

I think we have to check the Design but atm I have to time for that.

**Comment by steffenkleinle** [comment]
@hauf-toni is this actually `In Review`? Is this waiting for #2414?

**Comment by hauf-toni** [comment]
yes, it is not waiting for #2414 . please leave some feedback or alternatively implement these changes =)

**Comment by steffenkleinle** [comment]
> yes, it is not waiting for #2414 . please leave some feedback or alternatively implement these changes =)

Ok, thank you. I think this would need some changes on our side and we should try to make this uniform for all build configs (i.e. also for integreat, aschaffenburg and obdach) in a way that the logic is the same (but the specific colors obviously differ). Otherwise, we'd need a lot of logic dependent on the currently used build config.

One more question for understanding: How does this color change play together with the typography/design system you have planned? Won't there be another color change for buttons and so on? If yes, this feels a little redundant/unnecessary work.

**Comment by hauf-toni** [comment]
@steffenkleinle thanks for the clarification - we should put this topic on hold for a bit until the Integreat Design System is ready. We are currently working on the last 10%, so it shouldn't take too long. Then we can synchronize all the other build configurations.

**Comment by steffenkleinle** [comment]
@hauf-toni is this already done/addressed in the new design system? If yes, I'd close this issue as not planned.

**Comment by hauf-toni** [comment]
> [@hauf-toni](https://github.com/hauf-toni) is this already done/addressed in the new design system? If yes, I'd close this issue as not planned.

I would suggest we open another Issue for Applying the Design System to Malte (with adapted colors based on the CI color) and close this Issue.
